### PR TITLE
Add ARM image for CUDA

### PR DESCRIPTION
**Pull request by jakirkham**
Based off the existing ARM image and the PPC CUDA image, this adds a CUDA ARM image, which is needed for building CUDA ARM packages.

### CODE REVIEW

File: linux-anvil-aarch64-cuda/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+# Download and cache CUDA related packages.
+RUN source /opt/conda/etc/profile.d/conda.sh && \
+    conda activate && \
+    conda create -n test --yes --quiet --download-only \
+        conda-forge::cudatoolkit=${CUDA_VER}
+        && \
+    conda remove --yes --quiet -n test --all && \
+    conda clean -tiy && \
+    chgrp -R lucky /opt/conda && \
+    chmod -R g=u /opt/conda; \
```
Comment (jakirkham) [inline linux-anvil-aarch64-cuda/Dockerfile:101]: This should work after merging and building packages from PR ( https://github.com/conda-forge/cudatoolkit-feedstock/pull/59 )

**Comment by jakirkham** [comment]
cc @Ethyling (for awareness)

**Comment by jakirkham** [comment]
Rerunning CI now that PR ( https://github.com/conda-forge/cudatoolkit-feedstock/pull/59 ) is in and packages built

**Comment by jakirkham** [comment]
@conda-forge/core, this is ready for review! 😄

**Comment by leofang** [comment]
btw I noticed several CI jobs failed at the merge commit:
https://github.com/conda-forge/docker-images/actions/runs/1251380618

**Comment by jakirkham** [comment]
Yeah some of the `ppc64le` jobs failed ( https://github.com/conda-forge/docker-images/issues/190 )

**Pull request merged**